generate_postmortem.sh

```diff
@@ -670,6 +670,11 @@ if [[ $? -eq 0 && ${#OUTPUT} -gt 0 ]]; then
     echo "$OUTPUT" > "${K8S_CLUSTER_LIST_DATA}/icsp.yaml"
 fi
 
+#Get api-resources
+OUTPUT=`$KUBECTL api-resources 2>/dev/null`
+if [[ $? -eq 0 && ${#OUTPUT} -gt 0 ]]; then 
+    echo "$OUTPUT" > "${K8S_CLUSTER_LIST_DATA}/api-resources.out"
+fi
```

Comment:
in this particular case we are interested in any error that shows when running this command, so let's capture both stdout and stderr in the file unconditionally